core/exceptions/ExceptionStackTraceDisplay.php

```diff
@@ -77,7 +78,7 @@ protected function displayException(Exception $exception)
             echo wp_json_encode(array('error' => $styles . $output . $scripts));
             exit();
         }
-        echo $styles, $output, $scripts; // already escaped
+        echo $styles, wp_kses($output, AllowedTags::getWithFormTags()), $scripts;
```

Comment:
any time you find css or js that's being dumped directly to the DOM like this, plz create an issue for fixing it, thnx

Reply:
I created a new issue for this instance and I will add the rest to the created issue: https://github.com/eventespresso/event-espresso-core/issues/3839